Render/RenderSettings.as

```diff
@@ -2,7 +2,7 @@
 void RenderSettingsCustomization(){
 
     if(!UserCanUseThePlugin()){
-        UI::Text("You don't have the required permissions to use this plugin. You at least need the standard edition.");
+        UI::TextWrapped("You don't have the required permissions to use this plugin. You at least need the standard edition.");
```

Comment:
*The more you know*
Thanks a lot for this!

Reply:
Of course! Now if only I could find a way to limit the width of input fields as `SetNextItemWidth` doesn't seem to do that for me...